### PR TITLE
Remove previous ExPlat client

### PR DESCRIPTION
**Pull request by jessie-ross**
#### Changes proposed in this Pull Request

* Removes usages of the previous client
* Removes the ExPlatClient
* Changes the A/A over to the new client

Part of 579-gh-Automattic/experimentation-platform

#### Testing instructions

* Existing AT

### CODE REVIEW

**Comment by jessie-ross** [comment]
> Sections (~4185 bytes added 📈 [gzipped])

I am not sure why this has increased by 4kb?? Could it be the use of fragments?

**Review by mattgawarecki — APPROVED** [review]
Changes look good from Voyager's side. I was actually putting together a PR just now to remove our one existing test, since it's no longer running anyway. :-)

File: client/state/selectors/get-onboarding-url.js [hunk 1]
```diff
@@ -32,10 +31,7 @@ export default function getOnboardingUrl( state ) {
 		return config( 'gutenboarding_url' );
 	}
 
-	const existingUsersOnboardingVariant = getVariationForUser(
-		state,
-		'new_onboarding_existing_users_non_en_v5'
-	);
+	const existingUsersOnboardingVariant = null;
```
Comment (yanirs) [inline client/state/selectors/get-onboarding-url.js:34]: Lines 34-37 don't do anything useful now and line 36 is unreachable. Remove?

File: client/my-sites/plans/jetpack-plans/product-grid/index.tsx [hunk 2]
```diff
@@ -61,8 +59,7 @@ const ProductGrid: React.FC< ProductsGridProps > = ( {
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
 	const currentPlanSlug =
 		useSelector( ( state ) => getSitePlan( state, siteId ) )?.product_slug || null;
-	const exPlatVariation =
-		useSelector( ( state ) => getVariationForUser( state, SWITCH_PLAN_SIDES_EXPERIMENT ) ) || '';
+	const exPlatVariation = null;
```
Comment (yanirs) [inline client/my-sites/plans/jetpack-plans/product-grid/index.tsx:62]: This variable can be removed and `exPlatVariation === SWITCH_PLAN_SIDES_TREATMENT` below can be replaced with `false`.

**Review by yanirs — APPROVED** [review]
:tada: 

Can remove some more unused code, but other than that looks good to me.

File: client/state/selectors/get-onboarding-url.js (hunk 1 above)
Comment (jessie-ross) [reply client/state/selectors/get-onboarding-url.js:34]: I want to leave it up to experimenters to remove the actual experiment code.

**Pull request merged**